Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8/go-toolset:1.19 as builder
```

Comment:
BUILDPLATFROM is the host system platform, i.e. usually it's linux/amd64.

The current version of Dockerfile will create binaries for the host system, not the target system. Either you need to remove `--platform=$BUILDPLATFORM` and rely on emulation, or use TARGETARCH and TARGETOS and rely Go ability to cross-compile.

```
FROM --platform=$BUILDPLATFORM registry.access.redhat.com/ubi8/go-toolset:1.19 AS build
...
ARG TARGETOS TARGETARCH
RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -mod vendor -o manager main.go
```

See https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/ for details.

Reply:
Updated dockerfile to have `TARGETOS` and `TARGETARCH` as args

Reply:
@jcho02 need to remove `--platform=$BUILDPLATFORM`?

Reply:
no

Reply:
no